vanilla/locales
# Unhide English and British English Locales

English and British English locales are set to hidden, which means they do not appear on the locales dashboard menu and cannot be selected. This PR changed the 'Hidden' value to false, so that they appear.

### TO TEST
1. Go to the locale settings in the dashboard.
1. Verify that English and British English do not appear as selectable locales.
1. Check out this branch and verify that they do appear.
